### ISSUE TITLE
I want to grant other users edit access to my list

### ISSUE THREAD

**Issue by elordin** [riedmaph/elisa#32]
- prerequisite: I (the owning user) am logged in.

- [ ] in the list view i.e. where I can see the items of this specific list, there is the option to share this list with others
- [ ] selecting this option opens a view where I can see my last used contacts ( up to 5) and a field for entering a user name (i.e. the users log in email)
- [ ] Having entered the participants email the system shows whether this email adress is linked with a registered user. If that is the case, I can select him to share my list with.
- [ ] Selecting a user from recent contacts or after entering his email, I can choose to give edit or read access to him.
- [ ] selecting edit acess needs to be confirmed.
- [ ] after confirmation the new participant is registered as edit acces user for this list
- [ ] the new participant can see the list in his list overview
- [ ] the new particpant is able to see the items of this list.
- [ ] the new participant is able to move, enter, edit and complete items of that list

Grooming required: 
- ~Two types of acess (edit, read only) vs co-ownership~ resolved: currently only full access
- ~Notification for the new participant?~ Deferred until notifications are available 
- ~change the access type or remove the acess completly?~ Obsolete due to the point two above
- ~is the new user allowed to share this list or does that right belong to the owner?~ Obsolete due to the point two above

_Note for editing this story:_ keep in mind to update #31 accordingly if needed
